scss/_patterns_code-snippet.scss

```diff
@@ -63,5 +63,25 @@ $color-snippet-heading-bg: rgba($color-x-dark, 0.08);
       margin-bottom: 0;
       padding-top: 0;
     }
+
+    .p-code-snippet__dropdowns {
+      flex-shrink: 0;
+      margin-left: $sph-inner--small;
+    }
+
+    .p-code-snippet__dropdown {
+      background-color: $color-transparent;
+      border-color: $color-transparent;
+      color: $color-dark;
+
+      // taken from the .u-table-cell-padding-overlap utility
+      margin-bottom: -#{$table-cell-vertical-padding} !important;
+      margin-top: calc(-#{$table-cell-vertical-padding} - 1px) !important;
+      min-width: min-content;
+
+      // taken from the .is-dense button modifier
```

Comment:
Does this still apply when it's a select rather than a button?

Reply:
Yeah, I initially removed these when making the switch to selects, but there was way too much space around the element. Adding these back in solved that.